data/json/items/obsolete.json

```diff
@@ -517,5 +517,27 @@
         "volume": 1,
         "bashing": 4,
         "to_hit": -2
+    },
+    {
+        "id": "jar_V8",
```

Comment:
Consider a `migration` instead to a can of V8 as moving to `obsolete.json` does not prevent further references to the item existing and means code support must be maintained for any features (in this case the `use_action`).